### ISSUE TITLE
Feature: save configs in pretty format

### ISSUE THREAD

**Issue by jkulawik** [TiManGames/AmnesiaTheDarkDescent#21]
Would be nice if the configs saved with proper spacing rather than the very cramped XML we got today. Probably shouldn't be too complicated to implement

**Comment by jkulawik** [comment]
Sadly, it seems like this would require upgrading tinyXML and fixing the ensuing chaos of direct tinyXML usage and the cXmlDocumentTiny abstraction usage.

Looking at the source of tinyXML, it should save in pprint by default; in fact, we see this in save files for the game progress. But for no apparent reason, editor configs (which use `cConfigFile`) save unformatted. Perhaps tinyXML simply doesn't format long attribute.

Long story short, doing this is too involved for the tiny benefit it gives.
